app/assets/javascripts/respondent_detail.es6

```diff
@@ -0,0 +1,31 @@
+$(document).ready(() => {
+  if ($("#bail_conditions_set").length) {
```

Comment:
I suggest having this condition wrap all the code. Otherwise the functions will be created on all pages and could conflict with other functions. Especially `updateError` which has a rather generic name.